ads/dable.js

```diff
@@ -42,13 +42,16 @@ export function dable(global, data) {
   }
 
   const itemId = data['itemId'] || '';
+  const opts = {};
 
   if (itemId) {
     global.dable('sendLog', 'view', {id: itemId});
+  } else {
+    opts.ignoreItems = true;
   }
 
   // call render widget
```

Comment:
Do you have documentation on this that could benefit from noting this update? =)

Reply:
Oh, thanks for reply.

Before this update, page without `itemId` render dable widget within 3 seconds delay. After this update, the delay will be removed.